src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Task Class used to hold the job description as well as the status of the job.
+ */
+public class Task {
+    String job;
+    String status;
+
+    public Task(String job) {
+        this.job = job;
+        incomplete();
+    }
+
+    // Unicode representation of a tick
+    public void complete() {
+        this.status = "\u2713";
+    }
+    // Unicode representation of a cross
+    public void incomplete() {
```

Comment:
Nice that you use a function to set tick, but perhaps this function should be named setIncomplete() or setCross()?